runtime/common/src/main/kotlin/kotlinx/serialization/json/Json.kt

```diff
@@ -138,9 +162,9 @@ public class Json(
 
     companion object : StringFormat {
         val plain = Json()
-        val unquoted = Json(unquoted = true)
-        val indented = Json(indented = true)
-        val nonstrict = Json(strictMode = false)
+        val unquoted = Json(JsonConfiguration(unquoted = true, useArrayPolymorphism = true))
```

Comment:
Here we can either preserve semantics for default instances or break it and annotate them with `@UnstableDefault`. I'm not sure which way is better. Preserving semantics seems right to me since most users are not using any advanced features, but eventually, closer to the release, we still would need to change it.